pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -274,11 +274,11 @@ void run() throws PulsarAdminException {
         @Parameter(description = "tenant/namespace", required = true)
         private java.util.List<String> params;
 
-        @Parameter(names = "--subscription", description = "Subscription name for which permission "
+        @Parameter(names = {"-s", "--subscription"}, description = "Subscription name for which permission "
                 + "will be revoked to roles", required = true)
         private String subscription;
 
-        @Parameter(names = "--role", description = "Client role to which revoke permissions", required = true)
+        @Parameter(names = {"-r", "--roles"}, description = "Client role to which revoke permissions", required = true)
```

Comment:
Please keep the previous full name “--role”

Reply:
> Please keep the previous full name “--role”

sure，fixed